src/app/shared/navigation-focus/navigation-focus.ts

```diff
@@ -9,16 +11,29 @@ let lastTimeoutId = -1;
 export class NavigationFocus implements OnInit {
   @HostBinding('tabindex') role = '-1';
 
-  constructor(private el: ElementRef) {}
+  constructor(private el: ElementRef, private router: Router) {}
 
   ngOnInit() {
-    clearTimeout(lastTimeoutId);
-    // 100ms timeout is used to allow the page to settle before moving focus for screen readers.
-    lastTimeoutId = window.setTimeout(() => this.el.nativeElement.focus({preventScroll: true}),
-      100);
+    this.router.events
+      .pipe(filter((event: Event): event is RouterEvent => event instanceof RouterEvent))
+      .subscribe((routerEvent: RouterEvent) => {
+        // only want to focus on the element if the navigation was a soft nav,
+        // otherwise first focusable element should be the skip link
+        if (!(routerEvent.id === 0)) {
```

Comment:
can we make this `if(isSoftNav(routerEvent))`, easier to follow the logic that way